applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/server_store.h

```diff
@@ -0,0 +1,403 @@
+ * @return	-EALREADY Server is already assigned to the same conn.
+ * @retval	Other negative.	Errors from underlying functions.
+ */
+int srv_store_conn_update(struct bt_conn *conn, bt_addr_le_t const *const addr);
+
+/**
+ * @brief	Clear the contents of a server based on conn pointer.
+ *
+ * @note	This function can be used in a disconnect callback to clear all the contents of a
+ * server. Must only be called when there are no active connections. srv_store_lock() must be called
+ * before accessing this function.
+ *
+ * @return 0 on success, negative error code on failure.
+ */
+int srv_store_clear_by_conn(struct bt_conn const *const conn);
+
+/**
+ * @brief	Remove a single stored server based on conn pointer.
+ *
+ * @param[in] conn  Pointer to the connection associated with the server to remove.
+ *
+ * @note	Depending on the application, it is strongly recommended to call this function
+ * when an unbonded/untrusted connection is terminated or the bond for that connection is cleared.
+ * If not, the address will still be stored, and other/new connections which maliciously presents
+ * the same address will be recognized as a valid previously stored server.
+ * srv_store_lock() must be called before accessing this function.
+ *
+ * @return 0 on success, negative error code on failure.
+ * @return	-EACCES Server has active conn.
+ * @retval	Other negative.	Errors from underlying functions.
+ */
+int srv_store_remove_by_conn(struct bt_conn const *const conn);
+
+/**
+ * @brief	Remove a single stored server based on address.
+ * @param[in] addr  Pointer to the address associated with the server to remove.
+ *
+ * @note	srv_store_lock() must be called before accessing this function.
+ *
+ * @return 0 on success, negative error code on failure.
+ */
+int srv_store_remove_by_addr(bt_addr_le_t const *const addr);
+
+/**
+ * @brief	Remove all stored servers.
+ *
+ * @note	Must only be called when there are no active connections.
+ * srv_store_lock() must be called before accessing this function.
+ *
+ * @return 0 on success, negative error code on failure.
+ */
+int srv_store_remove_all(void);
+
+/**
+ * @brief Lock/take the server store semaphore.
+ *
+ * @param timeout	Waiting period to take the semaphore,
+ *			or one of the special values K_NO_WAIT and K_FOREVER.
+ * @param str		String to log the lock source.
+ *
+ * @retval 0 lock taken. One can now operate on the server store.
+ * @retval -EBUSY Returned without waiting.
+ * @retval -EAGAIN Waiting period timed out, or the semaphore was reset during the waiting period.
+ */
+int _srv_store_lock(k_timeout_t timeout, const char *file, int line);
+
+#ifdef CONFIG_DEBUG
+#define srv_store_lock(timeout) _srv_store_lock(timeout, __func__, __LINE__)
+#else
+#define srv_store_lock(timeout) _srv_store_lock(timeout, NULL, 0)
+#endif
+
+/**
+ * @brief Unlock/give the server store semaphore.
+ *
+ * @param str String to log the unlock source.
+ */
+void srv_store_unlock(void);
```

Comment:
```suggestion
/**
 * @brief Unlock/give the server store semaphore.
 */
void srv_store_unlock(void);
```
Would this work as a fix to:
```
server_store.h:392: argument 'str' of command @param is not found in the argument list of srv_store_unlock(void)
```